modules/apps/template/template-test/src/testIntegration/java/com/liferay/template/web/internal/portlet/action/test/CopyTemplateEntryMVCActionCommandTest.java

```diff
@@ -127,28 +126,34 @@ public void testCopyTemplateEntry() throws Exception {
 
 		Assert.assertTrue(templateEntries.size() > 1);
 
+		TemplateEntry templateEntry = null;
+
 		DDMTemplate originalDDMTemplate =
 			_ddmTemplateLocalService.getDDMTemplate(
 				_templateEntry.getDDMTemplateId());
 
-		Stream<TemplateEntry> templateEntriesStream = templateEntries.stream();
-
-		Assert.assertTrue(
-			templateEntriesStream.map(
-				templateEntry -> _ddmTemplateLocalService.fetchDDMTemplate(
-					templateEntry.getDDMTemplateId())
-			).filter(
-				Objects::nonNull
-			).filter(
-				ddmTemplate -> Objects.equals(
-					name, ddmTemplate.getName(languageId))
-			).filter(
-				ddmTemplate -> Objects.equals(
-					description, ddmTemplate.getDescription(languageId))
-			).anyMatch(
-				ddmTemplate -> Objects.equals(
-					originalDDMTemplate.getScript(), ddmTemplate.getScript())
-			));
+		for (TemplateEntry curTemplateEntry : templateEntries) {
+			DDMTemplate ddmTemplate = _ddmTemplateLocalService.fetchDDMTemplate(
+				curTemplateEntry.getDDMTemplateId());
+
+			Assert.assertNotNull(ddmTemplate);
```

Comment:
@JanisZhang The original `filter(Objects::nonNull)` should not be converted to `Assert.assertNotNull` here. The loop is to iterate through all the elements in `templateEntries` and see if there's any element meeting the conditions. If the first element in the list doesn't meet this condition, `Assert` will throw error, and the loop can not continue.

Also, no need to flip the condition below to shortcut, because the logic is pretty simple. Just use the combined condition, assign the variable and break in the if-body.